src/main/java/com/externalSorting/app/ExternalSortingRunner.java

```diff
@@ -9,60 +9,74 @@
     private final String originalFile;
     private final String outputFolder;
 
-    public ExternalSortingRunner(String originalFile, String outputFile) {
+    /**
+     * Constructor
+     *
+     * @param originalFile
+     * Path to the file that will be externally sorted
+     * @param outputFolder
+     * Path to the folder which will contain both the temporary files and the output file
+     */
+    public ExternalSortingRunner(String originalFile, String outputFolder) {
```

Comment:
I have a useful hint here - you can make a use of abstraction in order to reduce the amount of arguments in the constructor. You can create a new class called e.g. `PathsData` which will have 2 fields: `originalFilePath` and `outputFolderPath`. 
Then, in `main()` method, you would instantiate `PathsData` class, set `originalFilePath`, set `outputFolderPath` and pass it to `ExternalSortingRunner` constructor.
Abstracting parameters and objects passed into method/constructors like this is common approach.